tests/HDWalletTests.cpp

```diff
@@ -19,6 +20,86 @@
 
 namespace TW {
 
+const auto mnemonic1 = "ripple scissors kick mammal hire column oak again sun offer wealth tomorrow wagon turn fatal";
+const auto password = "password";
+
+#define EXPECT_EXCEPTION(statement, exceptionMsg) \
```

Comment:
this looks useful, maybe we can put it in a standalone header file

Reply:
Also planned to do it, I did it now.